include/mega/types.h

```diff
@@ -531,6 +532,23 @@ struct recentaction
 };
 typedef vector<recentaction> recentactions_vector;
 
+typedef enum
+{
+    ACCOUNT_TYPE_FREE = 0,
+    ACCOUNT_TYPE_PROI = 1,
+    ACCOUNT_TYPE_PROII = 2,
+    ACCOUNT_TYPE_PROIII = 3,
+    ACCOUNT_TYPE_LITE = 4,
+    ACCOUNT_TYPE_BUSINESS = 100
+} account_t;
+
+typedef enum
```

Comment:
Move the enumeration to the public interface (`MegaApi` or any suitable `MegaXXX` object).